pages/agent/v3/signed_pipelines.md

```diff
@@ -35,6 +35,9 @@ The following fields are included in the signature for each step:
 - **Matrix configuration.** The matrix configuration is signed as a whole rather than each individual matrix job. This means the signature is the same for each job in the matrix. When signatures are verified for matrix jobs, the agent double-checks that the job it received is a valid construction of the matrix and that the signature matches the matrix configuration.
 - **The repository the commands are running in.** This prevents you from copying a signed step from one repository to another.
 
+> 📘 Compatibility with [pipeline templates](/docs/pipelines/templates)
+> Due to the inclusion of repository in step signatures, signed steps cannot be used with pipeline templates. This is because templates are designed to be used across multiple pipelines, and therefore repositories.
```

Comment:
Maybe reverse the order of the sentences, and also avoid using fancy formatting and links for the callouts heading, which should just be sentence case without the final full-stop.

```suggestion
> 📘 Compatibility with pipeline templates
> [Pipeline templates](/docs/pipelines/templates) are designed to be used across multiple pipelines and therefore, repositories. Due to the inclusion of repositories in step signatures, signed steps cannot be used with pipeline templates.
```

Reply:
Thanks! Updated